src/main/java/seedu/address/model/association/exceptions/ClientPetAssociationListEmptyException.java

```diff
@@ -0,0 +1,7 @@
+package seedu.address.model.association.exceptions;
+
+/**
+ * Exception when the client list is empty and user tries to sort
```

Comment:
the header comment need not so specific to sort.
I think this exception can be used for other functions other than sort.

Reply:
alright will issue the amendments by tonight! 👍 thanks! 😄